elastic/kibana
# [Unified observability] Hide blocks from the overview page if the corresponding app is disabled in the space

## Summary

Closes #127444 

## Testing

- Create a space, with a bunch of observability sections disabled.
- Open that space
- Navigate to the overview page.

- [ ] You should not see the block for the disabled app.
- [ ] In the status page, the link to go to the app should be disabled (if the app has data)
